lib/nerves_network/config.ex

```diff
@@ -29,7 +29,7 @@ defmodule Nerves.Network.Config  do
 
   def init([]) do
     Logger.debug fn -> "#{__MODULE__}: init([])" end
-    SR.register(hysteresis: 50, min_interval: 500)
+    SR.register()
```

Comment:
how come? can we add some comment #

Reply:
This fixed a crash during start-up, I'm guessing the crash must've been caused by the IPv4 events then if we're missing them now.